app/furniture/marketplace/product.rb

```diff
@@ -4,6 +4,7 @@ class Marketplace
   class Product < Record
     include Archivable
 
+    # TODO: Refactor to use Media model
```

Comment:
I can create an issue and tag it here if that makes sense?

Reply:
Sure! Link it up as a 🧹to https://github.com/zinc-collective/convene/issues/1428!

Reply:
https://github.com/zinc-collective/convene/issues/2138